src/coreclr/pal/src/thread/process.cpp

```diff
@@ -1336,6 +1336,8 @@ static BOOL PROCEndProcess(HANDLE hProcess, UINT uExitCode, BOOL bTerminateUncon
             // abort() has the semantics that
             // (1) it doesn't run atexit handlers
             // (2) can invoke CrashReporter or produce a coredump, which is appropriate for TerminateProcess calls
+            // The code in the synchmanager.cpp line 1684 sets the exit code to this special value. The Watson analyzer
```

Comment:
Nit, since line numbers go stale a bit too fast
```suggestion
            // TerminationRequestHandlingRoutine in synchmanager.cpp sets the exit code to this special value. The Watson analyzer
```